pkg/features/kube_features.go

```diff
@@ -688,6 +688,14 @@ const (
 	// Decouples Taint Eviction Controller, performing taint-based Pod eviction, from Node Lifecycle Controller.
 	SeparateTaintEvictionController featuregate.Feature = "SeparateTaintEvictionController"
 
+	// owner: @aramase
+	// kep: https://kep.k8s.io/4412
+	// beta: v1.32
```

Comment:
don't duplicate the beta comment here any more, it's encoded in the versioned feature gate now